app/models/utility.rb

```diff
@@ -15,20 +15,21 @@
 #  jsonb                                :jsonb
 #  created_at                           :datetime         not null
 #  updated_at                           :datetime         not null
+#  content_length_limits                :jsonb
 #
 class Utility < ApplicationRecord
   include EntityWithCode
   include AttributeJsonParser
 
-  JSON_ATTRIBUTES = %i[integration_urls].freeze
+  JSON_ATTRIBUTES = %i[integration_urls content_length_limits].freeze
 
   has_many :users, dependent: :destroy
 
   validates :name, uniqueness: true
   validates :name, :type, presence: true
 
   store_accessor :integration_urls, :external_api_authentication_url, :books_data_url,
-                 :notes_data_url
+                 :notes_data_url, :content_length_limits, :short_content, :medium_content
```

Comment:
Entiendo que iría en un store_accessor diferente, porque el primer atributo es el nombre de la columna, algo asi:
```suggestion
                 :notes_data_url
   store_accessor :content_length_limits, :short_content, :medium_content
```

Reply:
Ahi cambie para que tener en distinto store accessor para separar la logica de las integraciones externas y la funcionalidad de cambiar la longitud de tipo de notas.